modules/UI/videolayout/VideoLayout.js

```diff
@@ -687,25 +687,21 @@ var VideoLayout = {
     /**
      * On last N change event.
      *
-     * @param lastNEndpoints the list of last N endpoints
-     * @param endpointsEnteringLastN the list currently entering last N
+     * @param endpointsLeavingLastN the list currently leaving last N
      * endpoints
      */
-    onLastNEndpointsChanged (lastNEndpoints, endpointsEnteringLastN) {
+    onLastNEndpointsChanged (endpointsLeavingLastN) {
```

Comment:
I haven't touched the logic about the UI that @yanas has added. And I think it is updated when actual UI is updated (largeVideo or thumbnail updateView).